backend/src/components/auth.js

```diff
@@ -32,14 +32,35 @@ const auth = {
 
   // Get new JWT and Refresh tokens
   async renew(refreshToken) {
+    log.info('renew called');
     let result = {};
 
     try {
+      //If local enviornment, use IDIR for both authentication strategies. (don't need to use business bceid)
+      let oidcClientId = null;
+      let oidcSecret = null;
+
+      if ('local' === config.get('environment')) {
+        oidcClientId = config.get('oidc:clientIdIDIR');
+        oidcSecret = config.get('oidc:clientSecretIDIR');
+      } else {
+        oidcClientId = config.get('oidc:clientId');
+        oidcSecret = config.get('oidc:clientSecret');
+      }
+
       const discovery = await utils.getOidcDiscovery();
+      log.info('discovery token endpoint: ', discovery.token_endpoint);
+      log.info('url: ', qs.stringify({
```

Comment:
querystring is deprecated. new code should use the `URLSearchParams` API instead.